package.json

```diff
@@ -41,7 +41,7 @@
     "clean:ljs": "pnpm turbo clean --filter=\"./libs/ledgerjs/**\"",
     "doc:ljs": "pnpm turbo doc --filter=\"./libs/ledgerjs/**\"",
     "watch:coin": "pnpm turbo run watch --filter=\"./libs/coin-*/**\" --concurrency 20",
-    "watch:es:coin": "pnpm turbo run watch:es --filter=\"./libs/coin-*/**\" --concurrency 20",
+    "watch:es:coin": "pnpm turbo run watch:es --filter=\"./libs/coin-*/**\" --concurrency 24",
```

Comment:
Why did we change it here?

Reply:
Since we added new coin-sui module we need to make 20+1. But for some reason `watch:es:coin` not starting until we make it 24.